### PR TITLE
Linted recipe_weapon.json Pt. 2

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Each chunk is 10 recipes.

### CODE REVIEW

File: data/json/recipes/recipe_weapon.json [hunk 1]
```diff
@@ -767,1090 +767,746 @@
...
-      [ "small_repairkit", 15 ],
-      [ "large_repairkit", 15 ]
-    ]
-  ],
-  "components": [
-    [
-      [ "scrap", 6 ],
-      [ "pipe", 1 ],
-      [ "steel_chunk", 2 ]
+  },  {
```
Comment (Night-Pryanik) [inline data/json/recipes/recipe_weapon.json]: Linter did this?

File: data/json/recipes/recipe_weapon.json [hunk 2]
```diff
@@ -767,1090 +767,746 @@
...
+        [ "welder_crude", 25 ],
+        [ "toolset", 25 ],
+        [ "small_repairkit", 15 ],
+        [ "large_repairkit", 15 ]
+      ]
     ],
-    [
-      ["flamethrower_crude", 1]
+    "components": [ [ [ "scrap", 6 ], [ "pipe", 1 ], [ "steel_chunk", 2 ] ], [ [ "plastic_chunk", 3 ] ], [ [ "spring", 2 ] ] ]
+  },  {
```
Comment (Night-Pryanik) [inline data/json/recipes/recipe_weapon.json]: And this.

File: data/json/recipes/recipe_weapon.json [hunk 3]
```diff
@@ -767,1090 +767,746 @@
...
+        [ "welder_crude", 25 ],
+        [ "toolset", 25 ],
+        [ "small_repairkit", 15 ],
+        [ "large_repairkit", 15 ]
+      ]
     ],
-    [
-      ["pipe", 1]
+    "components": [ [ [ "scrap", 6 ], [ "pipe", 1 ], [ "steel_chunk", 2 ] ], [ [ "spring", 1 ] ], [ [ "plastic_chunk", 3 ] ] ]
+  },  {
```
Comment (Night-Pryanik) [inline data/json/recipes/recipe_weapon.json]: And this.

File: data/json/recipes/recipe_weapon.json (hunk 1 above)
Comment (BorkBorkGoesTheCode) [reply data/json/recipes/recipe_weapon.json]: No. The linter doesn't support linting small sections of json without [] so each chunk had to be linted as a mini-file. I added the whitespace while adding and trimming [] characters and aligning the chunks.

**Pull request merged**